GitCommands/PathUtil.cs

```diff
@@ -72,20 +77,20 @@ public static bool IsLocalFile(string fileName)
         }
 
         /// <summary>
-        /// A naive way to check whether the given path is a URL by checking
-        /// whether it starts with either 'http', 'ssh' or 'git'.
+        /// A slightly less naive way to check whether the given path is a URL by checking
```

Comment:
It's unclear what this is less naive than. I mean, it makes sense in the diff, but won't in the resulting code.